backend/types.py

```diff
@@ -90,14 +90,34 @@ class LoadedDataPoint(DataPoint):
     )
 
 
+class ModelType(str, Enum):
+    """
+    Model types available in LLM gateway
+    """
+
+    completion = "completion"
+    chat = "chat"
+    embedding = "embedding"
+    
+class ModelConfig(BaseModel):
+    name: str
+    type: Optional[ModelType]
+    parameters: Optional[Dict[str, Any]] = None
+    
+class ModelProviderConfig(BaseModel):
+    provider_name: str
+    api_format: str
+    llm_model_ids: List[str]
+    embedding_model_ids: List[str]
+    api_key_env_var: str
```

Comment:
how is this `api_key` set?

Reply:
You can set the env variable for the key mentioned as the value here.